packages/checkout/sdk/src/smartCheckout/smartCheckout.ts

```diff
@@ -92,6 +93,7 @@ export const smartCheckout = async (
   routingOptions?: AvailableRoutingOptions,
   onComplete?: (result: SmartCheckoutResult) => void,
   onFundingRoute?: (fundingRoute: FundingRoute) => void,
+  fundingRouteFullAmount: boolean = true,
```

Comment:
In order to be backwards compatible with current behaviour do we need to default this to false? So implementation does fractional by default? Or am I getting the meaning mixed up hehe

Reply:
good catch, thanks Charlie